keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -794,6 +814,11 @@ This feature is implemented only in the API server and KCM and controlled by
 | off | on | external-provisioner/external-resizer should not get any event to create PVC with VolumeAttributesClass/update VolumeAttributesClass, the current resize flow stays the same |
 | on | on | New behavior.                                                                                                                    |
 
+After promotion to GA, the feature is enabled by default but can still be disabled. 
+Disabling is allowed because it could not be enabled by default during beta due to its 
+dependency on the off-by-default API group. Immediately enabling it by default with no 
+fallback option could be too risky.
```

Comment:
Can you add this reference page here? https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/5241-beta-featuregate-promotion-requirements#proposal

Reply:
Done